arangod/IResearch/IResearchViewCoordinator.cpp

```diff
@@ -273,6 +273,15 @@ Result IResearchViewCoordinator::appendVelocyPackImpl(
 }
 
 Result IResearchViewCoordinator::link(IResearchLink const& link) {
+#ifdef USE_ENTERPRISE
+  {
+    auto const& name = link.collection().name();
```

Comment:
If you don't mind I prefer to encapsulate filtering logic in a separate function, similar way you did for `createLink` https://github.com/arangodb/arangodb/blob/devel/arangod/IResearch/IResearchLinkHelper.cpp#L206

Reply:
Will do